.github/workflows/test.yml

```
@@ -0,0 +1,97 @@
+name: Tests
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)